pkgs/top-level/perl-packages.nix

```diff
@@ -6461,6 +6461,9 @@ let self = _self // overrides; _self = with self; {
     # Patch has been sent upstream.
     patches = [ ../development/perl-modules/gd-options-passthrough-and-fontconfig.patch ];
 
+    # otherwise "cc1: error: -Wformat-security ignored without -Wformat [-Werror=format-security]"
+    hardeningDisable = [ "format" ];
```

Comment:
Instead of disabling hardening, it also builds with
```NIX_CFLAGS_COMPILE = [ "-Wno-error=format-security" ];```
(but I didn't check if the package actually works).

Reply:
let's try